### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by arogl** [snejus/beetcamp#18]
Running the latest release, encountered the following error:

```
Traceback (most recent call last):
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/__init__.py", line 265, in handle
    return getattr(guru, attr)
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 526, in album
    filtered_tracks = self.tracks
  File "/home/arogl/.local/lib/python3.8/site-packages/cached_property.py", line 36, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 396, in tracks
    length=self.get_duration(raw_item),
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 154, in get_duration
    return [
IndexError: list index out of range
```

List of urls:
```
bandcamp: Unexpected error obtaining https://54thregiment.bandcamp.com/album/fall-in, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://alecsplatt.bandcamp.com/album/the-lost-album, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://chuckrocha.bandcamp.com/album/acoustic-hippie-punk, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://girlsinsynthesis.bandcamp.com/album/pressure, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://greenlung.bandcamp.com/album/black-harvest, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://hemorrhagingalienfetus.bandcamp.com/album/dont-you-forget-about-me, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://holdmyhand.bandcamp.com/album/a-grueling-summer, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://holdmyhand.bandcamp.com/album/a-grueling-summer, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://jgandmoriarty.bandcamp.com/album/songwriters, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://ladycatman.bandcamp.com/album/eyes-wide-open-2, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://loupgaroux.bandcamp.com/album/strangerlands, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://mavmusic.bandcamp.com/album/get-my-money, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://rossfromfriends.bandcamp.com/album/tread, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://seankinnear.bandcamp.com/album/senses, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://sensescoventry.bandcamp.com/album/little-pictures-without-sound, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://sensescoventry.bandcamp.com/album/little-pictures-without-sound, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://thedurites.bandcamp.com/album/pressure, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://theintuitionsmusic.bandcamp.com/album/fools-gold-2, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://theintuitionsmusic.bandcamp.com/album/fools-gold-2, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://themoose.bandcamp.com/album/the-moose-covers-the-best-of-tame-impala, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://thisheart.bandcamp.com/album/record-player, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://thisismusicbylily.bandcamp.com/album/wherever-you-go, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://windows-96.bandcamp.com/album/magic-peaks, please report at https://github.com/snejus/beetcamp/issues/new
bandcamp: Unexpected error obtaining https://yosoymusic.bandcamp.com/album/yosoy-music-presents-the-collection-1, please report at https://github.com/snejus/beetcamp/issues/new
```

**Comment by snejus** [comment]
I've had this before but could failed to figure out where exactly was it thrown out! 

Thanks for reporting, the fix is ready - will probably combine it with the rest

**Comment by snejus** [comment]
Fixed in `0.10.1`
